lib/mpool/src/mclass.c

```diff
@@ -407,7 +407,7 @@ void
 mclass_props_get(struct media_class *const mc, struct mpool_mclass_props *const props)
 {
     props->mc_fmaxsz = mblock_fset_fmaxsz_get(mc->mbfsp);
-    props->mc_mblocksz = mc->mblocksz >> MB_SHIFT;
+    props->mc_mblocksz = mc->mblocksz;
```

Comment:
Please double check that all the mclass_props_get() call sites does the necessary conversion of props->mc_mblocksz as the unit has changed.

Reply:
Why change it?  I don't think we ever want mblocks to be a fraction of 1M...

Reply:
I changed it because mpool receives the mblocksz from the params in bytes, so it made no sense to return mblocksz in MB.

Reply:
Probably specifying mblocksz in units of bytes is either wrong or not very useful (or both!), but we can address that some other time...